packages/pharos/src/components/image-card/pharos-image-card.scss

```diff
@@ -2,11 +2,13 @@
 
 :host {
   display: inline-flex;
+  width: 100%;
```

Comment:
I wasn't sure if this width should be included since we can apply it within the consuming app. But both `width: 100%` were necessary to get the right look and feel.

Reply:
Adding this here in a sense makes the card a block-level element, which to me seems like the consistent thing to do based on the desired behavior.